### ISSUE TITLE
Exception when used topic based on one partition

### ISSUE THREAD

**Issue by darrrrek** [jonathansant/Orleans.Streams.Kafka#7]
Hello,

This is my configuration of kafka stream provider:

```
.AddKafkaStreamProvider("KafkaStreamProvider", options =>
                {
                    options.BrokerList = new List<string> { "localhost:9092" };
                    options.ConsumerGroupId = "TestGroup";
                    options.Topics = new List<string> { "testtopic" };
                    options.PollTimeout = TimeSpan.FromMilliseconds(10);
                })
```

after application run, I get exception:

Exception while calling IQueueAdapter.CreateNewReceiver.
System.Collections.Generic.KeyNotFoundException: Key not found in dictionary.
   w System.ThrowHelper.ThrowKeyNotFoundException()
   w System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   w Orleans.Streams.Kafka.Core.KafkaAdapter.CreateReceiver(QueueId queueId)
   w Orleans.Streams.PersistentStreamPullingAgent.InitializeInternal(IQueueAdapt
er qAdapter, IQueueAdapterCache queueAdapterCache, IStreamFailureHandler failure
Handler)
fail: Orleans.Streams.PersistentStreamPullingManager-CommandsStreamProvider[1035
09]
      PersistentStreamPullingAgent testtopic-1805606425 failed to Initialize.
System.Collections.Generic.KeyNotFoundException: Key not found in dictionary.

What can be the reason ? all packages are updated to latest versions.

Thanks !
Darek

**Comment by jonathansant** [comment]
Hi, @darrrrek  is the topic `"testtopic"` created? Topics are not created automatically

**Comment by darrrrek** [comment]
HI @jonathansant 

Yes, this exception is thrown only if topic exists, if I'm using topic which not exist then exception is not thrown, very strange :)

**Comment by jonathansant** [comment]
Hi, @darrrrek thanks for the screenshot. I'll investigate between today and tomorrow. It's super odd since this is working in our production environment.

**Comment by darrrrek** [comment]
@jonathansant probably you are using topics based on more than 1 partitions :) in this case it's working for me also.

**Comment by jonathansant** [comment]
you're right @darrrrek. I'll roll at a fix for it. Thanks a lot for your pointing it out :)

**Comment by darrrrek** [comment]
@jonathansant  you are welcome, very helpful package for me, thanks :)

**Comment by jonathansant** [comment]
Should be fixed in `0.6.5` please let me know if you encounter any issues
